### PR TITLE
feat: use stable `tailwind` flag

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
See https://github.com/remix-run/remix/pull/5960

### CODE REVIEW

**Comment by runmoore** [comment]
Tested this change locally and it works perfectly 🥳

**Comment by machour** [comment]
@runmoore careful, you loose the autoprefixer that came free with tailwind

**Comment by runmoore** [comment]
> @runmoore careful, you loose the autoprefixer that came free with tailwind

Sorry, I'm not sure what you mean. Could you elaborate? I thought this would be a no brainier, maybe I'm wrong?

**Comment by machour** [comment]
@runmoore juste a little inconvenience, that could be solved manually. Check this for more context: https://github.com/remix-run/remix/issues/5475

**Comment by brophdawg11** [comment]
I think we should wait until future flags stabilize into a `v2_` flag before we auto-enable them in templates/stacks.  all the `unstable_` CSS stuff should be getting rolled up into `future.v2_dev` in `1.16.0` so let's hold until then?

**Comment by mcansh** [comment]
> you loose the autoprefixer that came free with tailwind

can we add a `postcss.config.js` file and `autoprefixer` as a devDep?
```js
module.exports = {
  plugins: [require("autoprefixer")],
};
```

**Comment by brophdawg11** [comment]
Just confirming, we have to wait until we release `1.16.0` before we can merge this, correct?

**Comment by MichaelDeBoey** [comment]
> we have to wait until we release `1.16.0` before we can merge this, correct?

@brophdawg11 That's correct.
Hence why I keep this in draft for now

**Review by machour — APPROVED** [review]

File: postcss.config.js
Comment (silvenon) [inline postcss.config.js:3]: Just wondering, why was tailwindcss added to the configuration as well? Wasn't the fix only to add autoprefixer?

**Pull request merged**
